modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -232,8 +233,22 @@ def find_by_ssn(ssn)
           # rubocop:enable Rails/DynamicFindBy
         end
 
+        def check_request_ssn_matches_mpi(req_headers)
+          req_ssn = req_headers['HTTP_X_VA_SSN']
+          ssn = auth_headers['va_eauth_pnid']
```

Comment:
Is it possible for the pnid to not be the SSN? Should we look at the `pnidtype` field?

Reply:
seems like it builds from the EVSS Auth Headers, which takes in the target_veteran - that should come from MPI. 

https://github.com/department-of-veterans-affairs/vets-api/blob/0f08a6f0b24175d7cf37ea0f1f5a63b831783ed0/modules/claims_api/app/controllers/claims_api/v1/forms/base.rb#L38

https://github.com/department-of-veterans-affairs/vets-api/blob/0f08a6f0b24175d7cf37ea0f1f5a63b831783ed0/lib/evss/auth_headers.rb#L29